src/main/java/seedu/address/logic/commands/ExportCommand.java

```diff
@@ -0,0 +1,46 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.io.File;
+import java.io.IOException;
+
+import seedu.address.model.Model;
+
+/**
+ * Exports addressbook.json into user's local device.
+ */
+public class ExportCommand extends Command {
+
+    public static final String COMMAND_WORD = "export";
+
+    public static final String MESSAGE_EXPORT_ACKNOWLEDGEMENT = "Exporting addressbook.json under "
+            + "file path data/addressbook.json in whichever folder addressbook.jar is in";
+    private static final String ADDRESSBOOK_FILE_PATH = "data/addressbook.json";
+
+    @Override
+    public CommandResult execute(Model model) {
+        requireNonNull(model);
+
+        try {
+            File file = new File(ADDRESSBOOK_FILE_PATH);
```

Comment:
I might be missing it but how does the JSON file get placed into the folder the addressbook,jar file is in?

Reply:
The path data/addressbook.json is relative to the location where addressbook.jar is run.